ftw/book/browser/resources/ftwbook.css

```diff
@@ -115,6 +115,36 @@ span.keyword, body.mceContentBody span.keyword {
   padding-right: 1px;
 }
 
+/** TINYMCE footnote plugin **/
+.ploneSkin span.mce_tinymce_footnote {
+  background: url("++resource++ftw.book.tinymce.footnote/img/footnote.png") no-repeat center center;
+}
+
+span.footnote-info {
```

Comment:
Could we maybe get rid of the nested `span.footnote-info` by using `:after`?
I think something like that should be possible:
```html
<span class="footnote" data-footnote="foo">bar</span>
```

```css
span.footnote:after {
  content: '';
  ....
  background-image: url(++resource++ftw.book.tinymce.footnote/img/info.png);
  ...
}
```